### PR TITLE
remove typo handling in get_axis_unit

### PR DESCRIPTION
**Pull request by linuxrider**
As astropy 5 can handle most of the typos which are included in the typos list we can omit the typos part in `get_axis_unit` and just print a nice message when astropy fails to convert the given string.

### CODE REVIEW

**Comment by linuxrider** [comment]
@saraedum I have tried to doctest the exception for a failing conversion by astropy. I could not get it to work so commented it out for now. 
Does it make sense to test the exception? If yes you can maybe help getting it to work.

File: svgdigitizer/electrochemistry/cv.py [hunk 1]
```diff
@@ -248,42 +248,20 @@ def get_axis_unit(cls, unit):
...
-                    return u.Unit(correct_unit)
+            >>> unit = 'uA cm⁻²'
+            >>> CV.get_axis_unit(unit)
+            Unit("uA / cm2")
 
-        raise ValueError(f"Unknown Unit {unit}")
+        """
+        #    >>> unit = 'milliV'
+        #    >>> CV.get_axis_unit(unit) # doctest: +IGNORE_EXCEPTION_DETAIL
+        #    ValueError: Astropy was not able to convert the given string (milliV) into a meaningful astropy unit. Please review the unit string.
```
Comment (saraedum) [inline svgdigitizer/electrochemistry/cv.py:258]:
```suggestion
        >>> unit = 'milliV'
        >>> CV.get_axis_unit(unit) # doctest: +IGNORE_EXCEPTION_DETAIL
        Traceback (most recent call last):
        ...
        ValueError: Astropy was not able to convert the given string (milliV) into a meaningful astropy unit. Please review the unit string.
```

Comment (saraedum) [inline svgdigitizer/electrochemistry/cv.py:258]:
```suggestion

        >>> unit = 'milliV'
        >>> CV.get_axis_unit(unit) # doctest: +IGNORE_EXCEPTION_DETAIL
        Traceback (most recent call last):
        ...
        ValueError: Astropy was not able to convert the given string (milliV) into a meaningful astropy unit. Please review the unit string.
        """
```

File: svgdigitizer/electrochemistry/cv.py [hunk 2]
```diff
@@ -248,42 +248,23 @@ def get_axis_unit(cls, unit):
...
+        Traceback (most recent call last):
+        ...
+        ValueError: Astropy was not able to convert the given string (milliV) into a meaningful astropy unit. Please review the unit string.
+        """
+        try:
+            return u.Unit(unit)
+        except ValueError as err:
+            raise ValueError(
+                f"Astropy was not able to convert the given string ({unit}) into a meaningful astropy unit. Please review the unit string."
+            ) from err
```
Comment (saraedum) [inline svgdigitizer/electrochemistry/cv.py:267]: Actually, the astropy exception is kind of nice so we might one to keep it?

```suggestion
        return u.Unit(u)
```

It says "'milliV' did not parse as unit: At col 0, milliV is not a valid unit. Did you mean milliVolt or millivolt? If this is meant to be a custom unit, define it with 'u.def_unit'. To have it recognized inside a file reader or other code, enable it with 'u.add_enabled_units'. For details, see https://docs.astropy.org/en/latest/units/combining_and_defining.html".

Sure, the col 0 is confusing and the final part is also weird. But the "did you mean" actually works here.

Comment (linuxrider) [reply svgdigitizer/electrochemistry/cv.py:267]: So, should we completley eliminate get_axis_unit()?

File: svgdigitizer/electrochemistry/cv.py [hunk 3]
```diff
@@ -248,42 +248,18 @@ def get_axis_unit(cls, unit):
...
-                if unit == typo:
-                    return u.Unit(correct_unit)
 
-        raise ValueError(f"Unknown Unit {unit}")
+        >>> unit = 'milliV'
+        >>> CV.get_axis_unit(unit) # doctest: +IGNORE_EXCEPTION_DETAIL
+        Traceback (most recent call last):
+        ...
+        ValueError: Astropy was not able to convert the given string (milliV) into a meaningful astropy unit. Please review the unit string.
+        """
```
Comment (linuxrider) [inline svgdigitizer/electrochemistry/cv.py:261]:
```suggestion

```

File: svgdigitizer/electrochemistry/cv.py [hunk 4]
```diff
@@ -248,42 +248,12 @@ def get_axis_unit(cls, unit):
...
-            "mA": ["mA", "milliampere"],
-            "A": ["A", "ampere", "amps", "amp"],
-            "mV": ["milliV", "millivolt", "milivolt", "miliv", "mV"],
-            "V": ["V", "v", "Volt", "volt"],
-            "V / s": ["V s-1", "V/s", "V / s"],
-            "mV / s": ["mV / s", "mV s-1", "mV/s"],
-        }
+            >>> unit = 'uA cm⁻²'
+            >>> CV.get_axis_unit(unit)
+            Unit("uA / cm2")
```
Comment (linuxrider) [inline svgdigitizer/electrochemistry/cv.py:254]:
```suggestion

        """
```

File: svgdigitizer/electrochemistry/cv.py [hunk 5]
```diff
@@ -248,42 +248,13 @@ def get_axis_unit(cls, unit):
...
 
-        for correct_unit, typos in unit_typos.items():
-            for typo in typos:
-                if unit == typo:
-                    return u.Unit(correct_unit)
+        """
 
-        raise ValueError(f"Unknown Unit {unit}")
+        return u.Unit(u)
```
Comment (linuxrider) [inline svgdigitizer/electrochemistry/cv.py:257]:
```suggestion
        return u.Unit(unit)
```

**Review by saraedum — APPROVED** [review]

**Pull request merged**